pandas/tseries/tests/test_base.py

```diff
@@ -2704,25 +2709,25 @@ def test_repeat(self):
         for res in [index.repeat(3), np.repeat(index, 3)]:
             tm.assert_index_equal(res, exp)
 
-    def test_nat(self):
-        self.assertIs(pd.PeriodIndex._na_value, pd.NaT)
-        self.assertIs(pd.PeriodIndex([], freq='M')._na_value, pd.NaT)
-
-        idx = pd.PeriodIndex(['2011-01-01', '2011-01-02'], freq='D')
-        self.assertTrue(idx._can_hold_na)
```

Comment:
Sorry about that forgot i commented those out when i was figuring out the error. 
I installed pandas on 32bit ubuntu, ran all the nosetests and keep getting that AssertionError.
Should i change it to intp or leave it?